experiments/prompt-injection-boundary-tags/rounds/round7/harness/run_experiment.py

```
@@ -0,0 +1,1412 @@
+        "generationConfig": {"temperature": 0.2},
+    }
+    if reasoning_budget in {"low", "high"}:
+        request_body["generationConfig"] = {
+            **dict(request_body.get("generationConfig", {})),
+            "thinkingConfig": {"thinkingBudget": _gemini_thinking_budget(reasoning_budget)},
+        }
+
+    headers = {"Content-Type": "application/json"}
+    try:
+        response_json = _json_post(url, headers, request_body)
+    except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)